airflow/providers/postgres/hooks/postgres.py

```diff
@@ -113,6 +114,18 @@ def schema(self):
     def schema(self, value):
         self.database = value
 
+    @property
+    def sqlalchemy_url(self) -> URL:
+        conn = self.get_connection(getattr(self, self.conn_name_attr))
+        return URL.create(
+            drivername="postgresql",
+            username=conn.login,
+            password=conn.password,
+            host=conn.host,
+            port=conn.port,
+            database=self.database or conn.schema,
```

Comment:
@Taragolis, I'm wondering if I should be passing query value by considering all the data passed via the `extra` field. Or the current implementation is fine?

Reply:
I'm sure this is fine because this one could be provided by two ways, from the Connection and directly from the hook